packages/runtime/test/src/parser/parser.ts

```diff
@@ -1977,7 +1980,11 @@ describe("Parser", function () {
 			});
 		});
 
-		describe("shoud return correct opcode for tealv6 ops", () => {
+		describe("should return correct opcodes for tealv6 ops", function () {
```

Comment:
Maybe something like this would be better

```suggestion
		describe("opcodes for tealv6 ops", function () {
```